src/lib.rs

```diff
@@ -29,13 +29,13 @@ enum RenderType {
 }
 
 impl RenderType {
-    fn render(self, img: &MinecraftSkin, size: u32, armored: bool) -> DynamicImage {
+    fn render(self, img: &MinecraftSkin, size: u32, armored: bool, slim: bool) -> DynamicImage {
```

Comment:
I'm starting to feel uncomfortable with this signature. At this point we should really pass around a `struct`:

```rust
struct RenderOptions {
    armored: bool,
    model: SkinModel,
}
```

Reply:
Yep, using a struct is a lot better, just implemented it.